RHDEVDOCS-3813 - added steps to install the operator using CLI (#42879)
openshift/openshift-docs: modules/installing-gitops-operator-using-cli.adoc

// Module is included in the following assemblies:
//
// * installing-red-hat-openshift-gitops

:_content-type: PROCEDURE
[id="installing-gitops-operator-using-cli_{context}"]
= Installing OpenShift GitOps Operator using CLI

.Procedure

. To install the Operator via the CLI, create a `Subscription`:
+
[source,yaml]
----
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-gitops-operator
  namespace: openshift-operators
spec:
  channel: stable
  installPlanApproval: Automatic
  name: openshift-gitops-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace 
----
. Apply the `Subscription` to the cluster:
+
[source,terminal]
----
$ oc apply -f openshift-gitops-sub.yaml
----

> . After the installation is complete, the following pods will appear:

Reviewer: @DebarghoGhosh I like the verification step :)

```suggestion
. After the installation is complete, ensure that all the pods in the `openshift-gitops` namespace are running:
```

> +

Reviewer: ```suggestion
+
.Example output
```

> [source,terminal]
> ----
> $ oc get pods -n openshift-gitops

Reviewer: ```suggestion
+
[source,terminal]
----
$ oc get pods -n openshift-gitops
----
```

> NAME                                                      	READY   STATUS	RESTARTS   AGE

Reviewer: ```suggestion
+
The output, which takes a few minutes to display, verifies that the {gitops-title} Operator is now installed in the `openshift-operators` default target namespace:
+
.Example output
[source,terminal]
----
NAME                                                      	READY   STATUS	RESTARTS   AGE
```

// Module is included in the following assemblies:
//
// * installing-openshift-gitops

:_content-type: PROCEDURE
[id="installing-gitops-operator-using-cli_{context}"]
= Installing OpenShift GitOps Operator using CLI

[role="_abstract"]
You can install {gitops-title} Operator from the OperatorHub using the CLI.

.Procedure

. Create a Subscription object YAML file to subscribe a namespace to the {gitops-title}, for example, `sub.yaml`:

.Example Subscription

> +

Reviewer: ```suggestion

```

[source,yaml]
----
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-gitops-operator
  namespace: openshift-operators
spec:
  channel: stable <1>
  installPlanApproval: Automatic
  name: openshift-gitops-operator <2>
  source: redhat-operators <3>
  sourceNamespace: openshift-marketplace <4> 
----
<1> Specify the channel name from where you want to subscribe the Operator.
<2> Specify the name of the Operator to subscribe to.
<3> Specify the name of the CatalogSource that provides the Operator.
<4> The namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.

> . Apply the `Subscription` to the cluster:

Reviewer: Suggestion: The output is not coming good. Please check and rectify. This is coming as step 1 and all the steps are spilling over to the right pane.
